.github/workflows/ci.yaml

```diff
@@ -3,7 +3,8 @@ on:
   pull_request:
   push:
     branches:
-      - master
+      - main
+      - devel
```

Comment:
why devel?

Reply:
I was going to say "for when we use `devel` branch but then you are going to say that "we should not give any feature that we don't use it because then becomes stable and we have to maintain it" and you are going to be right.

I am solving it in the next commit.